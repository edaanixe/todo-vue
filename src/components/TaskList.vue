<template>
    <div class="list-wrapper">
        <ul class="d-flex flex-column-reverse todo-list">
            <slot></slot>          
        </ul>
      </div>
</template>