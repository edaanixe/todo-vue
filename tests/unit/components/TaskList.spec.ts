import { shallowMount } from '@vue/test-utils'
import TaskList from '@/components/TaskList.vue'

describe('TaskList', () => {
    it('should match element against snapshot', () => {
        const wrapper = shallowMount(TaskList)
        expect(wrapper.element).toMatchSnapshot()
      })
});