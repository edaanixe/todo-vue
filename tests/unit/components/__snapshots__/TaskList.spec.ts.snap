// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TaskList should match element against snapshot 1`] = `
<div
  class="list-wrapper"
>
  <ul
    class="d-flex flex-column-reverse todo-list"
  />
</div>
`;
